python/apps/taiga/src/taiga/workflows/repositories.py

```diff
@@ -262,3 +262,16 @@ def get_status(
         return qs.get()
     except WorkflowStatus.DoesNotExist:
         return None
+
+
+##########################################################
+# WorkflowStatus - update workflow status
+##########################################################
+
+
+async def update_workflow_status(status: WorkflowStatus, values: dict[str, Any] = {}) -> WorkflowStatus:
+    for attr, value in values.items():
+        setattr(status, attr, value)
+
+    status.save()
```

Comment:
Yes, .asave() was introduced in Django 4.2, but I didn't manage to make it work
https://docs.djangoproject.com/en/4.2/ref/models/instances/#saving-objects

Let's see how it goes today with fresh eyes...

Reply:
It's because we are not in the Django 4.2 version yet. The other available async approaches I tried yesterday (mainly with `.aupdate()`/`aupdate_or_create()`) weren't so direct as "asaving()" the object.